doc/rst/technotes/gpu.rst

```diff
@@ -115,25 +117,35 @@ Requirements
   ``CHPL_LLVM`` must be set to ``system`` or ``bundled``). For more information
   about these settings see :ref:`Optional Settings <readme-chplenv>`.
 
-  * If using a ``system`` LLVM it must have been built with support for the
-    relevant target of GPU you wish to generate code for (e.g.  NVPTX to target
-    NVIDIA GPUs and AMDGPU to target AMD GPUs).
+  * If using a ``system`` LLVM:
+
+    * it must have been built with support for the relevant target of GPU you
+      wish to generate code for (e.g.  NVPTX to target NVIDIA GPUs and AMDGPU to
+      target AMD GPUs).
+
+    * we expect it to be the same version as the bundled version (currently 15).
+      Older versions may work; however, we only make efforts to test GPU support
+      with this version.
+
+* Either the CUDA toolkit (for NVIDIA), or ROCm (for AMD) must be installed
+  (unless using `CPU-as-Device mode`_)
 
-  * If using a system install of ``LLVM`` we expect this to be the same
-    version as the bundled version (currently 15). Older versions may
-    work; however, we only make efforts to test GPU support with this version.
+  * For targeting NVIDIA GPUs, we support CUDA versions from 10.x to 12.0
```

Comment:
Do you really mean `12.0` or do you mean `12.x`.

Our Python script would seem to let anything within 12.x through (https://github.com/chapel-lang/chapel/blob/624e5fabc5394fea6fe3f90a319d5e71ea45a929/util/chplenv/chpl_gpu.py#L211)

Reply:
Yeah, 12.x is probably better. I haven't tested beyond 12.0, but the "Tested Configurations" part should cover us there.